splash/browser_tab.py

```diff
@@ -129,10 +139,15 @@ def unsupported_content_handler(self, reply):
             
             fname = os.path.join(self.download_directory, fname)
             
-            with open(fname, 'wb+') as f: f.write(content)
-            
-            self.web_view.setHtml("<html><head /><body><h1>file was downloaded from %s to %s</h1></body>"%(url, fname))
+            # NOTE trying to retrieve the already downloaded chunks, cannot find the request in _response_bodies
+            #req_id = self.http_client.network_manager._get_request_id(reply.request())
+            #bodies = self.http_client.network_manager._response_bodies[req_id]
+            #log.msg('request id: %d, %d chunks' %(req_id, len(bodies)))
```

Comment:
I tried to retrieve the already downloaded chunks, but cannot find them in response_bodies